plugins/org.jkiss.dbeaver.data.office/src/org/jkiss/dbeaver/data/office/export/DataExporterXLSX.java

```diff
@@ -221,27 +224,27 @@ public void init(IStreamDataExporterSite site) throws DBException {
         styleHeader.setBorderRight(border);
 
         XSSFFont fontBold = (XSSFFont) wb.createFont();
-
+        fontBold.setFontName(fontName);
```

Comment:
What happens if you specify inexistent font? Don't you need to specify a fallback font in such cases?

It's rather a question than change request.